e2e/test/jest-react/package.json

```diff
@@ -52,28 +52,28 @@
   },
   "homepage": "https://github.com/reactstrap/reactstrap#readme",
   "dependencies": {
-    "react": "~16.13.1",
-    "react-dom": "~16.13.1",
-    "classnames": "~2.2.6",
+    "classnames": "~2.3.1",
     "prop-types": "~15.7.2",
-    "react-transition-group": "~4.4.1"
+    "react": "^16.14.0",
+    "react-dom": "^16.14.0",
+    "react-transition-group": "~4.4.2"
   },
   "peerDependencies": {
-    "react": "~16.13.1",
-    "react-dom": "~16.13.1"
+    "react": "~17.0.2",
+    "react-dom": "~17.0.2"
   },
   "devDependencies": {
-    "@babel/cli": "~7.10.5",
-    "@babel/core": "~7.11.4",
-    "@babel/plugin-proposal-export-default-from": "~7.10.4",
-    "@babel/plugin-proposal-export-namespace-from": "~7.10.4",
-    "@babel/plugin-proposal-object-rest-spread": "~7.11.0",
-    "@babel/plugin-transform-runtime": "~7.11.0",
-    "@babel/preset-env": "~7.11.0",
-    "@babel/preset-react": "~7.10.4",
+    "@babel/cli": "^7.14.8",
```

Comment:
Did you want to include this entire major version?

Reply:
sure, inside an e2e test 🤷‍♂️

Its probably not even used, so might want to remove it altogether, let me look

Reply:
Removed 🚮